tests/system/large/ml/test_linear_model.py

```diff
@@ -150,6 +150,44 @@ def test_logistic_regression_configure_fit_score(penguins_df_default_index, data
     assert reloaded_model.class_weight is None
 
 
+def test_unordered_mode_logistic_regression_configure_fit_score(
+    unordered_session, penguins_table_id, dataset_id
+):
+    model = bigframes.ml.linear_model.LogisticRegression()
```

Comment:
Not a problem, but if we only pick one model to test some shared functionalities, usually the choice is LinearReg.

Reply:
ok, used linear regression model instead